custom/icds_reports/management/commands/imbalanced_indicator.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import absolute_import, print_function
+
+from __future__ import unicode_literals
+
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+PERSON_TABLE_ID = 'static-person_cases_v2'
+AWC_LOCATION_TABLE_ID = 'static-awc_location'
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        indicator_list = self.get_indicator_list()
+
+        for indicator in indicator_list:
+            indicator_result = self.execute_query(indicator)
+
+            for ind in indicator_result:
+                print("{}, {}, {}, {}".format(ind[0], indicator['num'], ind[1], ind[2]))
+
+    def execute_query(self, params):
+        query = 'select month,({0}) as numerator, {1} as denomerator \
+        from {2} where  aggregation_level=1   group by month having ({0})>{1};'
```

Comment:
I think you just want to select for a single month (one that we can pass as a param), and then we could remove the `group by`, which might help performance.

Reply:
This would also be slightly better (at least for first pass), if you restricted to aggregation level 5 instead, because that would actually guarantee no indicators are above 100%. With level 1, some of the lower location levels may have indicators over 100% that we would not see.